### DISCUSSION TITLE
SMTP send Mail to multiple recipients

### DISCUSSION THREAD

**Discussion by sotzi** [Budibase/budibase#3843] [Help]
Is there a Solution to send an e-Mail to multiple recipients?

i have a form where I can choose multiple mail addresses to send to. If I choose only one address it works.

But if I add more than one, I receive an error!

**Comment by mjashanks** [comment]
Hmm, good idea. There is no way to do this at the moment. You might be able to use our REST API connector so send emails, if you are using a provider like SendGrid. We should enable this, don't think it would be difficult.

**Reply by sotzi** [reply]
Thanks for your answer!

It would be great when I could accomplish this with the normal smtp sending.

Meanwhile I’ll give a try with sendgrid. In an earlier version of budibase I have seen sendgrid as an automate option, but in the newest version it has been removed.

Does I have to create a Rest connection now?

**Reply by mjashanks** [reply]
Yes, we removed it when we added SMTP native to budibase. It didnt really make sense then. You should be able to do it with REST + an `External Data Connector` automation step.

**Reply by sotzi** [reply]
Hey, I am really sorry, but I am not able to create a connection to Sendgrid with the external Data Connector.
In can establish a connection, but I am not able to send the data fields to the api.

I am always gettin an error.

Could you please show mw how to do this?

Thank you.

**Reply by mjashanks** [reply]
What error are you getting?

**Comment by rohanrajpal** [comment]
Hey, is it possible to do this now? Would be super useful.

**Reply by melohagan** [reply]
I think this will be possible with automation looping in the upcoming release: https://docs.budibase.com/docs/automation-actions#looping

**Reply by rohanrajpal** [reply]
Interesting, looping would indeed be very helpful!

But I think it can be made possible with a single SMTP request. For example, in n8n, I send emails to multiple recipients via this method. 

![image](https://user-images.githubusercontent.com/7023147/165734420-3eddd166-bb5e-4904-ad0e-26920851d26b.png)

**Reply by charlfields** [reply]
> I think this will be possible with automation looping in the upcoming release: https://docs.budibase.com/docs/automation-actions#looping

I was able to use the looping with input type string, adding multiple emails addresses under the biding/value and {{ loop.currentItem }} under send to. Tested it, and it does indeed work this way. Thank you so much. 

Anyway https://github.com/Budibase/budibase/issues/4385#issue-1126984234 has requested this enhancement, other than adding the CC email, the looping already resolves this problem.

**Comment by giovannytj10** [comment]
Is there any way that when a ticket is created in which an email is added, that email is linked to the one that will be sent in the automation?
